test/algorithm/partial_sort_copy.cpp

```diff
@@ -41,7 +41,7 @@ test_larger_sorts(int N, int M)
     auto partial_sort_copy = ::make_testable_2<true, false>(
             [](auto&& ... args) {
                 return stl2::partial_sort_copy(
-                        std::forward<decltype(args)>(args)...);
+                        std::forward<decltype(args)>(args)...).out;
```

Comment:
Again, trailing `.whatever`. This time in tests, because it was the quickest way to make tests "work". This is spread out through all 3

Reply:
I did the same thing when I converted the first batch of algorithms to return `subrange` (e.g. rotate). It would be nicer to test both outputs of course, but this is fine with me for now.